marklogic-data-hub/src/main/java/com/marklogic/hub/impl/HubConfigImpl.java

```diff
@@ -155,6 +155,8 @@
     private String flowDeveloperUserName;
 
     private String dataHubAdminRoleName;
+    private String dataHubOperatorRoleName;
```

Comment:
We actually don't want these properties - we don't want to provide any ability to override these role names, as they're the agreed upon names that DHS will be expecting to use as well.

Reply:
now hardcoded